articles/enroll-hosts.md

```diff
@@ -10,15 +10,22 @@ Fleet supports the [latest version of osquery](https://github.com/osquery/osquer
 
 ## CLI
 
-> You must have `fleetctl` installed. [Learn how to install `fleetctl`](https://fleetdm.com/docs/using-fleet/fleetctl-cli#installing-fleetctl).
+> You must have `fleetctl` installed. [Learn how to install `fleetctl`](https://fleetdm.com/guides/fleetctl#installing-fleetctl).
 
 The `fleetctl package` command is used to generate Fleet's agent (fleetd) install package..
 
-The `--type` flag is used to specify the fleetd installer type. Note that Windows can only generate an MSI package:
+The `--type` flag is used to specify the fleetd installer type.
+
 - macOS: .pkg
-- Windows: .msi
+  - When building from Linux, you'll need Docker to run the [fleetdm/bomutils](https://hub.docker.com/r/fleetdm/bomutils) image that contains necessary dependencies.
+- Windows: .msi (using [WiX](https://github.com/wixtoolset/wix3))
+  - When building from Intel Macs or Linux, you'll need Docker to run the [fleetdm/wix](https://hub.docker.com/r/fleetdm/wix) image that contains the necessary WiX utilities.
+  - When building from Windows, by default `fleetctl` will use Docker, but you can [use WiX without Docker instead](https://fleetdm.com/guides/enroll-hosts#generating-fleetd-for-windows-using-local-wix-toolset).
+  - When building from Apple Silicon Macs, you'll need to [install Wine](https://fleetdm.com/install-wine), as `fleetctl` uses a local installation of WiX.
```

Comment:
```suggestion
  - Generating a .pkg on Linux requires [Docker](https://docs.docker.com/get-docker) to be installed and running.
- Windows: .msi (using [WiX](https://github.com/wixtoolset/wix3))
  - Generating a .msi on Windows, Intel Macs, or Linux requires [Docker](https://docs.docker.com/get-docker) to be installed and running. On Windows, you can [use WiX without Docker instead](https://fleetdm.com/guides/enroll-hosts#generating-fleetd-for-windows-using-local-wix-toolset).
  - Generating a .msi on Apple Silicon Macs requires [Wine](https://fleetdm.com/install-wine) to be installed.
```

I think for most users, they just want to know what dependencies they need to install how to get them. If I'm understanding correctly, for most of these, the user just has to worry about installing Docker.

Also using "generating" instead of "building" to be consistent w/ rest of the page.

Reply:
Deferring to @nonpunctual on desired wording here, as this [Slack conversation](https://fleetdm.slack.com/archives/C019WG4GH0A/p1727365659692609) seems to be what initiated this bug report.

Reply:
[This MacAdmins conversation](https://macadmins.slack.com/archives/C0214NELAE7/p1724187179300569) is the other place the missing docs got brought up.

Reply:
> - macOS: .pkg
>   - Generating a .pkg on Linux requires [Docker](https://docs.docker.com/get-docker) to be installed and running.
> - Windows: .msi
>   - Generating a .msi on Windows, Intel Macs, or Linux requires [Docker](https://docs.docker.com/get-docker) to be installed and running. On Windows, you can [use WiX without Docker instead](https://fleetdm.com/guides/enroll-hosts#generating-fleetd-for-windows-using-local-wix-toolset).
>   - Generating a .msi on Apple Silicon Macs requires [Wine](https://fleetdm.com/install-wine) to be installed.

@nonpunctual if your had the language I'm proposing here when you were trying to create fleetd for Windows, would you have what you need?

I'm trying to cut the language down to as few words as possible.